### PR TITLE
Core/PacketIO: update & enable CMSG_QUESTGIVER_HELLO

### PR DESCRIPTION
**Pull request by Luzifix**

### CODE REVIEW

**Comment by Luzifix** [comment]
is something wrong with the opcode update?

File: src/server/game/Server/Protocol/Opcodes.h [hunk 1]
```diff
@@ -725,7 +725,7 @@ enum OpcodeClient : uint32
     CMSG_TAXIENABLEALLNODES                           = 0xBADD,
     CMSG_TAXISHOWNODES                                = 0xBADD,
     CMSG_TAXI_NODE_STATUS_QUERY                       = 0x0EA1,
-    CMSG_TAXI_QUERY_AVAILABLE_NODES                   = 0x0B2A,
+    CMSG_TAXI_QUERY_AVAILABLE_NODES                   = 0xBADD,
```
Comment (MitchesD) [inline src/server/game/Server/Protocol/Opcodes.h]: why?

File: src/server/game/Server/Protocol/Opcodes.h [hunk 2]
```diff
@@ -1042,7 +1042,7 @@ enum OpcodeServer : uint32
     SMSG_CORPSE_TRANSPORT_QUERY                       = 0x0B6C,
     SMSG_CREATE_SHIPMENT_RESPONSE                     = 0xBADD,
     SMSG_CREATURE_QUERY_RESPONSE                      = 0x0DC2,
-    SMSG_CRITERIA_DELETED                             = 0x0B2A,
+    SMSG_CRITERIA_DELETED                             = 0xBADD,
```
Comment (MitchesD) [inline src/server/game/Server/Protocol/Opcodes.h]: again, why?

File: src/server/game/Server/Protocol/Opcodes.h (hunk 1 above)
Comment (Luzifix) [inline src/server/game/Server/Protocol/Opcodes.h]: because a opcode can only use one time

File: src/server/game/Server/Protocol/Opcodes.h (hunk 2 above)
Comment (Luzifix) [inline src/server/game/Server/Protocol/Opcodes.h]: because a opcode can only use one time

Comment (Ladel) [inline src/server/game/Server/Protocol/Opcodes.h]: I think you can put identical opcodes SMSG and CMSG

Comment (MitchesD) [inline src/server/game/Server/Protocol/Opcodes.h]: CMSG != SMSG

Comment (Ladel) [inline src/server/game/Server/Protocol/Opcodes.h]: yes

Comment (Luzifix) [inline src/server/game/Server/Protocol/Opcodes.h]: ups sorry, i change it after the work :) but CMSG_TAXI_QUERY_AVAILABLE_NODES is correct edit.

Comment (MitchesD) [inline src/server/game/Server/Protocol/Opcodes.h]: where did you find your opcode number?

Comment (Luzifix) [inline src/server/game/Server/Protocol/Opcodes.h]: Sniffing

**Comment by Luzifix** [comment]
Create new PR because, i dont cant squatch my other commit =/
https://github.com/TrinityCore/TrinityCore/pull/14391

**Pull request closed without merging**